keyboards/maxr1998/pulse4k/rules.mk

```
@@ -0,0 +1,33 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader
+#     This definition is optional, and if your keyboard supports multiple bootloaders of
+#     different sizes, comment this out, and the correct address will be loaded
+#     automatically (+60). See bootloader.mk for all options.
+BOOTLOADER = atmel-dfu
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
```

```suggestion
BOOTMAGIC_ENABLE = lite    # Virtual DIP switch configuration(+1000)
```
If your physical reset switch isnt easily accessible e.g hidden inside a case it may be worth setting this to lite for easier resetting else it can be ignored

---

Alright, thanks for the info! I think that'd make sense, then I don't have to swap it over all the time.